pages/dkp/kommander/1.3/release-notes/1.3.0/index.md

```diff
@@ -26,7 +26,9 @@ Kommander provides a command center for all your cloud native management needs i
 | **Maximum**        | 1.19.x  |
 | **Default**        | 1.19.7  |
 
-# Known issue
+# Known issues
+
+In Kommander 1.3.2 and below, Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that requiire a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the [upgrade instructions in the 1.3.3 release notes](../1.3.3/).
```

Comment:
FYI - just pushing this change - if you have big changes you'll need to pull first @gracedo 

```suggestion
In Kommander 1.3.2 and below, Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the [upgrade instructions in the 1.3.3 release notes](../1.3.3/).
```